paper/references.bib

```diff
@@ -742,4 +742,77 @@ @misc{bbchallenge_bb5
+  booktitle = {SAT},
+  pages     = {428--437},
+  year      = {2018},
+  url       = {https://doi.org/10.1007/978-3-319-94144-8_26},
+  doi       = {10.1007/978-3-319-94144-8_26}
+}
+
+@inproceedings{itk-sat24,
+  author    = {Alexey Ignatiev and Zi Li Tan and Christos Karamanos},
+  title     = {Towards Universally Accessible {SAT} Technology},
+  booktitle = {SAT},
+  pages     = {4:1--4:11},
+  year      = {2024},
+  url       = {https://doi.org/10.4230/LIPIcs.SAT.2024.16},
+  doi       = {10.4230/LIPICS.SAT.2024.16},
+}
+
+@inproceedings{DBLP:conf/ijcai/AudemardS09,
+  author    = {Gilles Audemard and Laurent Simon},
+  title     = {Predicting Learnt Clauses Quality in Modern SAT Solvers},
+  booktitle = {Proceedings of the 21st International Joint Conference on Artificial Intelligence, IJCAI 2009},
+  pages     = {399--404},
+  year      = {2009},
+  url       = {https://www.ijcai.org/Proceedings/09/Papers/074.pdf},
+  timestamp = {Mon, 08 Feb 2021 14:54:15 +0100},
+  biburl    = {https://dblp.org/rec/conf/ijcai/AudemardS09.bib},
+  bibsource = {dblp computer science bibliography, https://dblp.org}
+}
+
+@inproceedings{DBLP:conf/cp/AudemardS12,
+  author       = {Gilles Audemard and
+                  Laurent Simon},
+  editor       = {Michela Milano},
+  title        = {Refining Restarts Strategies for {SAT} and {UNSAT}},
+  booktitle    = {Principles and Practice of Constraint Programming - 18th International
+                  Conference, {CP} 2012, Qu{\'{e}}bec City, QC, Canada, October
+                  8-12, 2012. Proceedings},
+  series       = {Lecture Notes in Computer Science},
+  volume       = {7514},
+  pages        = {118--126},
+  publisher    = {Springer},
+  year         = {2012},
+  url          = {https://doi.org/10.1007/978-3-642-33558-7\_11},
+  doi          = {10.1007/978-3-642-33558-7\_11},
+  timestamp    = {Thu, 30 May 2024 21:47:23 +0200},
+  biburl       = {https://dblp.org/rec/conf/cp/AudemardS12.bib},
+  bibsource    = {dblp computer science bibliography, https://dblp.org}
+}
+
+@inproceedings{DBLP:conf/sat/EenS03,
+  author       = {Niklas E{\'{e}}n and
+                  Niklas S{\"{o}}rensson},
+  editor       = {Enrico Giunchiglia and
+                  Armando Tacchella},
+  title        = {An Extensible SAT-solver},
+  booktitle    = {Theory and Applications of Satisfiability Testing, 6th International
+                  Conference, {SAT} 2003. Santa Margherita Ligure, Italy, May 5-8, 2003
+                  Selected Revised Papers},
+  series       = {Lecture Notes in Computer Science},
+  volume       = {2919},
+  pages        = {502--518},
+  publisher    = {Springer},
+  year         = {2003},
+  url          = {https://doi.org/10.1007/978-3-540-24605-3\_37},
+  doi          = {10.1007/978-3-540-24605-3\_37},
+  timestamp    = {Tue, 14 May 2019 10:00:41 +0200},
+  biburl       = {https://dblp.org/rec/conf/sat/EenS03.bib},
+  bibsource    = {dblp computer science bibliography, https://dblp.org}
+}
```

Comment:
Several references are added to references.bib but as far as I understand are not cited in the text.  Is this wanted?

Reply:
Yes, but maybe @dsm054 was planning to use them and forgot, so it seemed worth pointing out.